.github/workflows/ci-llama-large-tests.yaml

```diff
@@ -68,7 +68,13 @@ jobs:
       - name: Run llama tests
         run: |
           source ${VENV_DIR}/bin/activate
-          pytest sharktank/tests/models/llama/benchmark_amdgpu_test.py -v -s --run-nightly-llama-tests --iree-hip-target=gfx942 --iree-device=hip://0 --html=out/llm/llama/benchmark/index.html
+          pytest sharktank/tests/models/llama/benchmark_amdgpu_test.py \
+            -v -s \
+            -m "expensive" \
```

Comment:
Thanks for splitting these out across multiple lines. Some of these small NFC changes would be useful to merge first in a smaller PR.